definitions/variable/energy/energy-final.yaml

```
@@ -15,416 +15,540 @@
     energy source: all
     energy use: all excl. losses
     sector: all end-users
-    unit: EJ/yr
+    unit: 
+    - EJ
+    - GWh
```

Same for all suggested changes
```suggestion
    unit: [EJ/yr, GWh]
```

---

Thank you Daniel, I will add this suggested change.